sourcecode/utilities.py

```diff
@@ -0,0 +1,44 @@
+MONTHS = ['','January','February','March','April','May','June','July','August','September','October','November','December']
+MON = ['','Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']
+def getWeatherDic (PATH):
```

Comment:
The function name should be in lower case with underscore separated.

Reply:
PEP8 violation, there should be no space between function names and arguments. The argument name should also be in lower case.

Reply:
Function names should be meaningful. Avoid using word which represents data structure or reserved names.